mistral_example.py

```diff
@@ -17,18 +20,18 @@
 model_path = 'mistralai/Mistral-7B-Instruct-v0.1'
 config = transformers.AutoConfig.from_pretrained(model_path)
 config.sliding_window = 200000000 # disable mistral's default SWA mechanism (4096), mistral's true window is 8192.
-model = AutoModelForCausalLM.from_pretrained(model_path, config=config, device_map="auto")
+model = AutoModelForCausalLM.from_pretrained(model_path, config=config, device_map=device)
 
 
 tokenizer = AutoTokenizer.from_pretrained(model_path)
-model.eval()
+model.eval().to(device)
 
 # In the example task file, the passkey is placed within the last 4096 tokens, this means, if you use SWA, mistral will successfully find the passkey.
 for line in open("passkey_examples_10k.jsonl", "r"):
     example = json.loads(line)
     prompt_postfix = "What is the pass key? The pass key is "
     prompt = example["input"] + prompt_postfix
-    input_ids = tokenizer(prompt, return_tensors="pt").input_ids
+    input_ids = tokenizer(prompt, return_tensors="pt").input_ids.to(device)
```

Comment:
device_map value "auto" can't be used in tensor.to(device) or model.eval().to(device). Valid device values are cpu, cuda, ipu, xpu, mkldnn, opengl, opencl, ideep, hip, ve, fpga, ort, xla, lazy, vulkan, mps, meta, hpu, mtia, privateuseone.